backend/app/api/routes/model_management.py

```
@@ -0,0 +1,263 @@
+    model_name: str,
+    model_config: ModelConfig = Body(...),
+    _current_user: User = Depends(get_current_user),
+):
+    """
+    动态更新模型配置
+    """
+    try:
+        # 先删除旧配置
+        try:
+            await _call_litellm_admin_api("POST", "/model/delete", {"id": model_name})
+        except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)